src/verifyUtils.ts

```diff
@@ -461,7 +442,7 @@ export async function verifyVC(
     const proofs: any = vc.proof;
     if (!proofs.length) {
         let hashes =
-            proofs.type === 'CordSDRProof2024' ? proofs.hashes : undefined;
+            proofs.type === 'CordSDRProof2025' ? proofs.hashes : undefined;
```

Comment:
It would be great to keep the old one as is, and handle this as a separate section.

Ie, I would like to make sure we verify the old one also properly if we get to 'Verify' service.